kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -733,7 +777,7 @@ public Properties getConsumerProperties() {
     addTelemetryReporterProperties(consumerProps);
     consumerProps.putAll(originalsWithPrefix("client.", /* strip= */ true));
     consumerProps.putAll(originalsWithPrefix("consumer.", /* strip= */ true));
-    consumerProps.putAll(originalsWithPrefix("schema.registry", /* strip= */ false));
+    consumerProps.putAll(getSchemaRegistryConfigs());
```

Comment:
same comment as producer.

Reply:
See comment on producer.

Reply:
Consumer is unchanged because it has not been rewritten. The consumer still needs the schema registry configs.